algorithms/unit_tests/CMakeLists.txt

```diff
@@ -24,7 +24,7 @@ KOKKOS_ADD_TEST_LIBRARY(
 # avoid deprecation warnings from MSVC
 TARGET_COMPILE_DEFINITIONS(kokkosalgorithms_gtest PUBLIC GTEST_HAS_TR1_TUPLE=0 GTEST_HAS_PTHREAD=0)
 
-IF(NOT (Kokkos_ENABLE_CUDA AND WIN32))
+IF((NOT (Kokkos_ENABLE_CUDA AND WIN32)) AND (NOT ("${KOKKOS_CXX_COMPILER_ID}" STREQUAL "Fujitsu")))
```

Comment:
Elaborate on why this exception is here in a comment, please.

Reply:
because cmake is weird and I need to do this to make it work?

Reply:
This can't be just a CMake problem. @jjwilke or is it?

Reply:
it kinda is, because it combines requesting a c++ standard via compile feature (instead of just setting a cxx flag) with checking whether the compiler actually supports it, which it comes back as negative if it doesn't identify the compiler as a known thing (like nvcc on windows, or fujitsu in gener). I guess I can add that as a comment.